### PR TITLE
Publish solutions for lab 7

### PR DESCRIPTION
**Pull request by andreia-oca**

### CODE REVIEW

File: labs/lab07/skel/1-custom/hash.c [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /**
- * SO, 2017 - Lab #07, Profiling
+ * SO
+ * Lab #07
+ *
```
Comment (sketch873) [inline labs/lab07/skel/1-custom/hash.c]: Trailing space

File: labs/lab07/skel/2-major/column.c [hunk 2]
```diff
@@ -1,5 +1,7 @@
 /**
- * SO, 2017 - Lab #07, Profiling
+ * SO
+ * Lab #07
+ *
```
Comment (sketch873) [inline labs/lab07/skel/2-major/column.c]: Trailing space

File: labs/lab07/skel/2-major/row.c [hunk 3]
```diff
@@ -1,5 +1,7 @@
 /**
- * SO, 2017 - Lab #07, Profiling
+ * SO
+ * Lab #07
+ *
```
Comment (sketch873) [inline labs/lab07/skel/2-major/row.c]: Trailing space

File: labs/lab07/skel/3-busy/busy.c [hunk 4]
```diff
@@ -1,16 +1,20 @@
 /**
- * SO, 2017 - Lab #07, Profiling
+ * SO
+ * Lab #07
+ *
```
Comment (sketch873) [inline labs/lab07/skel/3-busy/busy.c]: Trailing space

File: labs/lab07/skel/4-find-char/find-char.c [hunk 5]
```diff
@@ -1,5 +1,7 @@
 /**
- * SO, 2017 - Lab #07, Profiling
+ * SO
+ * Lab #07
+ *
```
Comment (sketch873) [inline labs/lab07/skel/4-find-char/find-char.c]: Trailing space

File: labs/lab07/skel/5-print/print.c [hunk 6]
```diff
@@ -1,5 +1,7 @@
 /**
- * SO, 2017 - Lab #07, Profiling & Debugging
+ * SO
+ * Lab #07
+ *
```
Comment (sketch873) [inline labs/lab07/skel/5-print/print.c]: Trailing space

File: labs/lab07/skel/6-flowers/flowers.c [hunk 7]
```diff
@@ -1,5 +1,7 @@
 /**
- * SO, 2017 - Lab #07, Profiling & Debugging
+ * SO
+ * Lab #07
+ *
```
Comment (sketch873) [inline labs/lab07/skel/6-flowers/flowers.c]: Trailing space

File: labs/lab07/skel/7-exploit/exploit.c [hunk 8]
```diff
@@ -1,5 +1,7 @@
 /**
- * SO, 2017 - Lab #07, Profiling & Debugging
+ * SO
+ * Lab #07
+ *
```
Comment (sketch873) [inline labs/lab07/skel/7-exploit/exploit.c]: Trailing space

**Comment by sketch873** [comment]
Can you also move make a `lin` directory in `skel` and `sol` and move the skeleton and solutions in them?

**Review by sketch873 — APPROVED** [review]

**Review by lucigrigo — APPROVED** [review]
Looks good.

**Pull request merged**